core/src/main/java/org/bitcoinj/evolution/SimplifiedMasternodeListManager.java

```diff
@@ -236,7 +237,7 @@ public void processMasternodeListDiff(@Nullable Peer peer, SimplifiedMasternodeL
         log.info("processing mnlistdiff between : " + quorumState.getMnList().getHeight() + " & " + newHeight + "; " + mnlistdiff);
         lock.lock();
         try {
-            quorumState.applyDiff(peer, headersChain, blockChain, mnlistdiff, isLoadedFromFile());
+            quorumState.applyDiff(peer, headersChain, blockChain, mnlistdiff, isLoadingBootStrap);
```

Comment:
`isLoadedFromFile()` was incorrectly specified

Reply:
The idea of the bootstrap is that a file that contains all masternodes and quorums can be loaded from the APK of an app to avoid downloading the entire data.  Instead later, when the client needs to update the masternodes/quorums, it will only need to download the difference compared to the bootstrap.